libs/lib_loaders.py

```diff
@@ -23,13 +25,13 @@
 from lib_db_obfuscator import db_hlapi
 from lib_sonnetconfig import CLIB_LOAD, GLOBAL_PREFIX, BLACKLIST_ACTION
 
-from typing import Dict, List, Union, Any, Tuple, Optional, Type, cast
+from typing import Any, Tuple, Optional, Union, cast, Type, Dict
```

Comment:
what the hell happened here?

Reply:
unimported 3.9 things, then reimported them when it broke